### PR TITLE
chore: add boilerplate files

### LINKED ISSUE #1: add boilerplate files to repo
Files like GH actions, gitignore, readme, etc

### PR DESCRIPTION
**Pull request by korikuzma**
close #1

### CODE REVIEW

**Comment by korikuzma** [comment]
Ope forgot to request review. This one is lower priority than the cat-vrs-python PR

**Review by jsstevenson — APPROVED** [review]
would generally just echo comments from other PR

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+# ANN102 - missing-type-cls
+# S101 - assert
+# B011 - assert-false
+"tests/*" = ["ANN001", "ANN2", "ANN102", "S101", "B011"]
+
+[tool.setuptools.packages.find]
+where = ["src"]
+
+[tool.setuptools_scm]
+
```
Comment (jsstevenson) [inline pyproject.toml:141]: extremely nitpicky, but can we move these up to be directly below the build system key?

**Review by jsstevenson — APPROVED** [review]
👍

**Pull request merged**